src/random_main.cpp

```diff
@@ -45,12 +46,14 @@ int pandora_random_path(RandomOptions const& opt)
     BOOST_LOG_TRIVIAL(info) << "Index loaded successfully!";
 
     Fastaq fa(opt.compress, false);
+    std::random_device random_dev;
+    std::mt19937 rng(random_dev());
```

Comment:
So can you not set a seed for the random subcommand?

Reply:
Indeed: https://github.com/rmcolq/pandora/issues/341